# any Touch velocity reading less than this (in absolute value) is discarded for noise reduction
min_velocity : 0.020
# this scale multiplies the velocities created on the touch
movement_scale : 0.5
